### ISSUE TITLE
.json badge errors

### ISSUE THREAD

**Issue by RedSparr0w** [badges/shields#1348]
## JSON + Style
If you try to use a `.json` badge and specify `?style=flat` or similar the server throws an error and it can occasionally crash the server.

output:
```
[server] 1206210259 Vendor hook error: TypeError: template is not a function
[server]     at makeBadge (C:\Users\Dan\Documents\GitHub\shields\lib\make-badge.js:151:10)
[server]     at sendBadge (C:\Users\Dan\Documents\GitHub\shields\lib\request-handler.js:192:23)
[server]     at C:\Users\Dan\Documents\GitHub\shields\server.js:813:7
[server]     at Request.request [as _callback] (C:\Users\Dan\Documents\GitHub\shields\lib\request-handler.js:165:9)
[server]     at Request.self.callback (C:\Users\Dan\Documents\GitHub\shields\node_modules\request\request.js:186:22)
[server]     at emitTwo (events.js:125:13)
[server]     at Request.emit (events.js:213:7)
[server]     at Request.<anonymous> (C:\Users\Dan\Documents\GitHub\shields\node_modules\request\request.js:1163:10)
[server]     at emitOne (events.js:115:13)
[server]     at Request.emit (events.js:210:7)
```
## JSON Static Badge
If you try using `.json` on a static badge `/badge/label-value-blue.json` it returns a 404 badge.
_no console errors thrown_
```json
{
  "name": "404",
  "value": "badge not found"
}
```

**Comment by espadrine** [comment]
I feel like the static badge issue is not too important. I cannot think of a use-case, since it would give the data that is fed untouched.

Nice find for the json+style error!

**Comment by paulmelnikow** [comment]
Fixed in #1429?
